core/lib/spree/testing_support/dummy_app.rb

```diff
@@ -18,6 +18,9 @@
 # @private
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+  def spree_current_user
```

Comment:
When using `try_spree_current_user` you can get away with not setting up any authentication and it will just return `nil`. By not using it we're relying more on the expectation on `ApplicationController` to implement this method. For test purposes having a no-op method is equivalent of using `try_spree_current_user` without an auth implementation.